.github/workflows/sdk-publish.yaml

```diff
@@ -268,3 +279,43 @@ jobs:
           footer: "Linked Repo <{repo_url}|{repo}> | <{run_url}|View Run>"
         env:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+  publish-gems:
+    if: ${{ needs.generate.outputs.ruby_regenerated == 'true' && inputs.publish_ruby == 'true' }}
+    name: Publish Ruby SDK
+    runs-on: ubuntu-latest
+    needs: release
+    defaults:
+      run:
+        working-directory: ${{ needs.generate.outputs.ruby_directory }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
```

Comment:
the `with` section is needed here

Reply:
I don't understand what you mean by this?